cmd/run.go

```diff
@@ -404,14 +404,17 @@ a commandline interface for interacting with it.`,
 					break mainLoop
 				}
 
+				hint := "You can increase the time limit via the "
```

Comment:
I would say this should be inside the `case lib.TimeoutError`

Reply:
Something like `TimeoutError.Hint` method?

Reply:
I meant that it should be a few lines below as it's only relevant in the `case` where we have `lib.TimeoutError` but I guess it can be a method of the `TimeoutError` as well